src/fluree/db/query/fql/parse.cljc

```diff
@@ -20,13 +20,9 @@
 
 (defn parse-context
   [{:keys [opts] :as q} db]
-  (let [ctx-key (if (= :string (or (:context-type opts)
-                                   (:contextType opts)))
-                  :context-str
-                  :context)
-        db-ctx  (get-in db [:schema ctx-key])
-        q-ctx   (or (:context q) (get q "@context"))]
-    (json-ld/parse-context db-ctx q-ctx)))
+  (dbproto/-context db
+                    (or (:context q) (get q "@context"))
+                    (:context-type opts)))
```

Comment:
The reason I excluded it is that it will only ever be needed for Clojure... so you'd never call it from JavaScript where camelCase makes sense. Because all context is only string-based, it wouldn't even make sense to open up this 'option' to JS, or JSON-based interaction.

But for consistency perhaps it should be camelCase as well.

Reply:
That makes sense. I don't have strong feels about including it here, really. My original thought was just that the spec in the `syntax` ns includes camel case, so one way or another the spec and the implementation should be in sync.